### PR TITLE
feat: OpenId Federation with multiple layers

### PR DESCRIPTION
**Pull request by Tommylans**

### CODE REVIEW

File: agent/package.json [hunk 1]
```diff
@@ -24,19 +24,6 @@
...
-      "@sphereon/did-auth-siop": "https://gitpkg.vercel.app/animo/OID4VC/packages/siop-oid4vp?funke",
-      "@sphereon/jarm": "https://gitpkg.vercel.app/animo/OID4VC/packages/jarm?funke",
-      "@sphereon/oid4vc-common": "https://gitpkg.vercel.app/animo/OID4VC/packages/common?funke",
-      "@credo-ts/askar": "https://gitpkg.vercel.app/animo/aries-framework-javascript/packages/askar?funke",
-      "@credo-ts/core": "https://gitpkg.vercel.app/animo/aries-framework-javascript/packages/core?funke",
-      "@credo-ts/node": "https://gitpkg.vercel.app/animo/aries-framework-javascript/packages/node?funke",
-      "@credo-ts/openid4vc": "https://gitpkg.vercel.app/animo/aries-framework-javascript/packages/openid4vc?funke",
-      "dcql": "0.2.17",
-      "@sphereon/pex-models": "2.3.2"
-    }
```
Comment (TimoGlastra) [inline agent/package.json:40]: this is imporatnt for the deployment (as it only has this package json, not the pnpm workspace) should match with the override in root packagej.son

File: agent/src/endpoints.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ import {
   getJwkFromKey,
 } from '@credo-ts/core'
 import { OpenId4VcVerificationSessionState } from '@credo-ts/openid4vc'
-import express, { type NextFunction, type Request, type Response } from 'express'
+import express, { response, type NextFunction, type Request, type Response } from 'express'
```
Comment (TimoGlastra) [inline agent/src/endpoints.ts:14]: i don't think the response import here is correct.

File: app/components/VerifyTab.tsx [hunk 3]
```diff
@@ -1,18 +1,20 @@
...
+  const [x509Certificate, setX509Certificate] = useState<string>()
+
+  useEffect(() => {
+    getX509Certificate().then(({ certificate }) => setX509Certificate(certificate))
+  }, [])
 
-  return <VerifyBlock flowName="Verify" createRequest={createRequestForVerification} />
+  return (
+    <>
+      <VerifyBlock flowName="Verify" createRequest={createRequest} x509Certificate={x509Certificate} />
```
Comment (TimoGlastra) [inline app/components/VerifyTab.tsx:17]: x509 cert was removed from verify block

**Review by TimoGlastra — APPROVED** [review]

File: agent/package.json (hunk 1 above)
Comment (Tommylans) [reply agent/package.json:40]: Ah I got a warning that is wasn't necessary I'll place it back

**Pull request merged**